source/MaterialXFormat/Util.cpp

```diff
@@ -36,6 +36,19 @@ string readFile(const FilePath& filePath)
     return EMPTY_STRING;
 }
 
+void getSubdirectories(const FilePathVec rootDirectories, const FileSearchPath& searchPath, FilePathVec& subdirectories)
```

Comment:
Simple recursive search across "roots". Each root can be to the root of a library for instance.

Reply:
As mentioned earlier we should probably use a manifest file for this, but this is fine as a work in progress.

Reply:
Yes. Agreed. Need to have the discussion on what this looks like first :)